kumascript/macros/HTTPSidebar.ejs

```diff
@@ -291,93 +312,84 @@ var text = mdn.localStringMap({
     <li class="section"><%-web.smartLink(`/${locale}/docs/Web/HTTP`, null, text['HTTP'])%></li>
     <li class="section no-link"><%=text['Guides']%></li>
     <li class="toggle">
-        <details>
-            <summary><%=text['ResourcesURI']%></summary>
-            <ol>
-                <li><%-web.smartLink(`/${locale}/docs/Web/URI`, null, text['Identifying'])%></li>
-                <li><%-web.smartLink(`/${locale}/docs/Web/URI/Schemes/data`, null, text['DataURLs'])%></li>
-                <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Basics_of_HTTP/MIME_types`, null, text['MIMETypes'])%></li>
-                <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types`, null, text['ListMIMETypes'])%></li>
-                <li><%-web.smartLink(`/${locale}/docs/Web/URI/Authority/Choosing_between_www_and_non-www_URLs`, null, text['WWWorNotWWW'])%></li>
-            </ol>
-        </details>
+      <details>
+        <summary><%=text['Concepts']%></summary>
+        <ol>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Overview`, null, text['Overview'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Messages`, null, text['Messages'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Headers`, null, text['Headers'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Methods`, null, text['Methods'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Status`, null, text['Status'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Session`, null, text['Session'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Compression`, null, text['Compression'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Content_negotiation`, null, text['ContentNego'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Proxy_servers_and_tunneling`, null, text['Proxies'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Connection_management_in_HTTP_1.x`, null, text['Connection1x'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Protocol_upgrade_mechanism`, null, text['ProtocolUpgradeMech'])%></li>
+          <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Evolution_of_HTTP`, null, text['Evolution'])%></li>
```

Comment:
Can we remove all the smartLink parameters except the first one? It'll automatically draw `short-title || title` from the page.

Reply:
Do you mean all smartLinks should have the following format?

```
<ol>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Overview`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Messages`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Headers`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Methods`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Status`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Session`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Compression`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Content_negotiation`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Proxy_servers_and_tunneling`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Connection_management_in_HTTP_1.x`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Protocol_upgrade_mechanism`)%></li>
  <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Evolution_of_HTTP`)%></li>
</ol>
```

Reply:
Yes, exactly. This is how it will be in the new sidebar architecture.

Reply:
@caugner do you mind having another look?

Reply:
Looking good!